BotModules/SteamBot/index.js

```diff
@@ -19,22 +23,19 @@ module.exports = function(bot) {
     console.info(`Steambot logged in as ${bot.user.tag}!`);
   });
 
-  function pullMentions(action){
-    return function(msg) {
-      let arrMentions = msg.mentions.users.map(user => user.id)
-      if(arrMentions.length > 1){
-        action(msg, arrMentions);
-      }
```

Comment:
Not sure if this was intended or a consequence of merging, but pullMentions is no longer here. Just wanted to draw attention to this.

Reply:
I made pull mentions before you had done your project to solve the issue of the command requiring 2 arguments. You fixed that in a different way on your branch so decided to keep with that approach so when I merged I pulled the code since your solution did the same job.